frontend/packages/dev-console/integration-tests/features/getting-started/getting_started _to_dev_perspective.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Create Sample Application
+    As a user I want to create the Sample Application from Add options
```

Comment:
```suggestion
    As a user, I want to create the Sample Application from +Add page
```

Reply:
If you are okay then can you please update that sentence to: 
```suggestion
    As a user, I should be able to create the Sample Application from +Add page
```

Reply:
`I want` has been used multiple times in other files and I think it is fine to use it here

Reply:
`want` is used in multiple files and it fits correctly so IMO there is no need to change it